microsetta_public_api/server.py

```diff
@@ -39,9 +47,11 @@ def run(app):
     if use_test_database:
         # import TestDatabase here to avoid circular import
         from microsetta_public_api.utils.testing import TestDatabase
-        with TestDatabase():
-            app = build_app()
-            run(app)
+        tdb = TestDatabase()
+        tdb.start()
+        app = build_app()
+        run(app)
+        tdb.stop()
```

Comment:
should `stop` be registered with `atexit` to assure it's called?

Reply:
Yeah probably if you don't want a bunch junk files floating around on your system 😅 

Thanks for letting me know that exists!

Reply:
Done in https://github.com/biocore/microsetta-public-api/pull/55/commits/4e13c62aa60bb29bc296c27b2d57afe35db0ef00